vm/src/builtins/singletons.rs

```diff
@@ -52,6 +54,13 @@ impl PyNone {
     }
 }
 
+impl AsNumber for PyNone {
+    const AS_NUMBER: PyNumberMethods = PyNumberMethods {
+        boolean: Some(|number, _vm| Ok(self::number_downcast(number).value.is_zero()))
```

Comment:
is_zero? the value looking swapped

Reply:
`non_bool()`, which is of `none_as_number`, returns 0 in cpython `Objects/object.c`
```
static int
none_bool(PyObject *v)
{
    return 0;
}
```

Reply:
return zero means false in this case. As our `bool` returns false as you know, we just need to return what `bool` returns exactly.

Reply:
oh, sorry. I was confused with bool. 
as you said, it returns false(0). But this code is not simply returning false. make this line to do return false same to the code you shared.

Reply:
Hi. I fixed my code. Could you please review the new commit?